adagucserverEC/CDBFileScanner.cpp

```diff
@@ -734,7 +734,6 @@ int CDBFileScanner::DBLoopFiles(CDataSource *dataSource, int removeNonExistingFi
                       if (dimVar->getType() == CDF_STRING) {
                         const char *str = ((char **)dimVar->data)[i];
                         uniqueKey.print("%s", str);
-                        uniqueKey.replaceSelf(" ", "_");
                         uniqueKey.replaceSelf("<", "_lt_");
```

Comment:
You want to keep the replacements of <, > and = ? They are probably used nowhere but if used by accident they could cause problems?

Reply:
Jep because these endup in the GetCapabilities and cause issues